### PR TITLE
#166840878 landing page

### PR DESCRIPTION
**Pull request by olorunwalawrence**
#### What does this PR do?
- create mobile auth page
#### Description of Task to be completed?
- create mobile auth  page

#### Screenshots
Screenshot 2019-08-03 at 3 47 55 PM

### CODE REVIEW

**Review by Onyimatics — APPROVED** [review]
Good job @olorunwalawrence .Looks good to me. I hope you have made the changes requested by TTL?

**Review by timi-codes — CHANGES_REQUESTED** [review]
Good Job @olorunwalawrence, However, I think you should rename the file and components. Registration gives the impression that only the signup button is on the page. Maybe you could try something around AuthMenu.

**Review by kechyy — APPROVED** [review]
Good Job @olorunwalawrence . Looking exactly like the mock up design.

File: src/routes/AppRouter.js [hunk 1]
```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LandingPage from '../pages/loginPage';
 import Homepage from '../pages/hompage';
+import Card from '../components/registration/index';
```
Comment (tolumide-ng) [inline src/routes/AppRouter.js]: you might want to remove index from the end of this so you can have something like `import Card from '../components/registration` instead

File: src/routes/AppRouter.js [hunk 2]
```diff
@@ -2,11 +2,13 @@ import React from 'react';
...
 
 const Routes = () => (
   <Switch>
     <Route path="/" exact component={Homepage} />
     <Route path="/login" exact component={LandingPage} />
+    <Route path="/card" exact component={Card} />
```
Comment (tolumide-ng) [inline src/routes/AppRouter.js]: I am still unsure if this is the right way this should work, how does this component know it should be rendered when the screensize is small? and not render when the screen size is large

File: src/components/registration/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const cards = () => {
+  return (
+    <div className="flex justify-center w-full md:hidden ">
+      <div className="max-w-sm flex items-center h-screen overflow-hidden w-full">
+        <div className="flex justify-center items-center h-screen flex-col w-full">
+          <div className="">
+            <button
+              type="submit"
```
Comment (tolumide-ng) [inline src/components/registration/index.js]: `type` should be button and not `submit`, there is no form at the top and make a dummy link of where this should link to. Can we do the same thing for the get started button to?

File: tests/reg.spec.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Jest from 'jest';
+import toJson from 'enzyme-to-json';
+import { shallow } from './enzyme';
+import Registration from '../src/components/registration/index';
+
+
+describe('Application test', () => {
+  it('should work fine on App', () => {
+    const wrapper = shallow(<Registration />);
```
Comment (tolumide-ng) [inline tests/reg.spec.js]: Good job @olorunwalawrence , can we change this to `mount`? Use `shallow` when you are rendering a React component within the component that you are shallowing

**Review by tolumide-ng — CHANGES_REQUESTED** [review]
Great job @olorunwalawrence , please check my Reviews and rebase

File: tests/auth.spec.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { mount } from './enzyme';
+import Auth from '../src/components/auth';
```
Comment (Onyimatics) [inline tests/auth.spec.js:4]: Please can you change this `Auth` to something like `AuthLogin`.

File: src/components/auth/index.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+          <div className="flex flex-col items-center">
+            <a
+              href="/login"
+              className="bg-teal-500  text-white text-center text-xs w-32  uppercase  py-3 px-5 "
+            >
+              LOGIN
+            </a>
+
+            <a 
+              href="/login"
```
Comment (Onyimatics) [inline src/components/auth/index.js]: `Get started` should have a `signup` link and not `login`

**Review by Onyimatics — CHANGES_REQUESTED** [review]
Well-done @olorunwalawrence . I left some comments.

**Review by timi-codes — APPROVED** [review]
Looks good to me! @olorunwalawrence

**Pull request merged**